chapter6_Fruitful_Functions/example.py

```diff
@@ -0,0 +1,51 @@
+# Return Values
+import math
+
+
+def area(radius):
+    a = math.pi * radius**2
+    return a
+
+def absolute_value(x):
+    if x < 0:
+        return -x
+    else:
+        return x
+    
+
```

Comment:
For now, the space between two functions should be 1 line.
Please double-check for all.